README.md

```diff
@@ -59,8 +58,16 @@ or install top level dataset by omitting `-r datasets` option and then calling
 
 You can navigate datasets you have installed in your terminal or browser,
 while fetching necessary files or installing new sub-datasets using the
-same `datalad install [FILENAMES]` command.
-
+`datalad get [FILE|DIR]` command.  DataLad will take care about
```

Comment:
- yes -- it should support multiple unless there is a really good reason not to
- `[FILE|DIR]` means 1 file or a directory, so I don't think this one needs tune up